etc/kayobe/ansible/cardiff-run.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Cardiff run
+  hosts: localhost
+  gather_facts: no
+  tags:
+    - cardiff
+  vars:
+    venv: "~/venvs/cardiff-venv"
+    input_dir: "{{ lookup('env', 'PWD') }}/overcloud-introspection-data"
+    output_dir: "{{ lookup('env', 'PWD') }}/review"
+    cardiff_pattern: ".*.eval" # Uses regex
+  tasks:
+    - name: Install dependencies
+      pip:
+        virtualenv: "{{ venv }}"
+        name:
+          - git+https://github.com/stackhpc/cardiff
+          #- /home/matt/Documents/cardiff
+          - pyvis
+          - pyvis.network
+        state: latest
+
+    - name: Create data directory
+      file:
+        path: '{{ output_dir }}/data'
+        state: directory
+
+    - name: Extract data
+      shell:
+        cmd: >
+          {{ venv }}/bin/m2-extract {{ input_dir }}/*.json --output_dir {{ output_dir }}/data
+
+    - name: Create review directory
+      file:
+        path: '{{ output_dir }}/results'
+        state: directory
+
+    - name: Process data
+      shell:
+        cmd: >
+          {{ venv }}/bin/cardiff
+          -I ipmi
+          -p '{{ output_dir }}/data/extra-hardware/{{ cardiff_pattern }}'
+          -o '{{ output_dir }}/results'
+          --visualise
+        #  --visualise_dataless
```

Comment:
Are there various options that may be useful here? Could make it a variable:

```
cardiff_opts: "--visualise"
```

Reply:
Decided the dataless option was unnecessary, have removed it.